src/NextPage.js

```diff
@@ -0,0 +1,59 @@
+import { useParams } from "react-router-dom";
+import { useEffect, useState } from "react";
+import axios from "axios";
+import { Link } from "react-router-dom";
+
+// anime={anime} nextPage={nextPage} setNextPage={setNextPage} setAnime={setAnime}
+
+const NextPage = ({ nextPage, setAnime, anime, setNextPage }) => {
+
+    const { page } = useParams();
+
+    useEffect( () => {
+        axios({
+        url: 'https://api.jikan.moe/v4/top/anime',
+        method: 'GET',
+        dataResponse: 'json',
+        params: {
+          page: page
+        }
+      }).then((res) => {
+        const results = res.data.data;
+        setAnime(results);
+      })
```

Comment:
Not sure if this is completely necessary, but if you have time it doesn't hurt. 
Add a `.catch()` like with how you did in `AboutAnime.js`'s axios call. Maybe like an `alert` in the case that there's a 404 response?